pom.xml

```diff
@@ -89,9 +89,6 @@
       <email>antoon.johansson@gmail.com</email>
       <timezone>+1</timezone>
     </contributor>
-    <contributor>
-      <name>Andrzej Jarmoniuk</name>
-    </contributor>
```

Comment:
@jarmoniuk 
Did you forget add yourself to developers list? 
If it is your intention to remove at all that ok.